docs/integrations/app-integrations/web-crawler.mdx

```diff
@@ -69,4 +74,22 @@ WHERE url = '<link-to-pdf-file>'
 LIMIT 1;
 ```
 
-For example, you can provide a link to a `pdf` file stored in Amazon S3.
+## Troubleshooting
```

Comment:
Do we need to add information about bypassing captcha and other bot protection software? I don't believe our handler has the ability to do this?

Reply:
We don't want to do that for now; it raises a few legal issues. From an ethical perspective, maybe we need to check the site robot.txt rules.